jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Session.java

```diff
@@ -1647,4 +1589,149 @@ private void complete()
             terminate(failure);
         }
     }
+
+    /**
+     * SPEC: It is required that stream ids are strictly crescent.
+     * Here we use a queue to atomically create the stream id and
+     * claim the slot in the queue. Concurrent threads will only
+     *
```

Comment:
Remove extra line, and link the spec section. Also, finish the comment: "Concurrent thread will only..." 
I know it's my stuff and I left it incomplete, but please complete it :smiley:

Reply:
sure.